src/main.cpp

```diff
@@ -1,9 +1,17 @@
 #include "database/database.h"
+#include "ui/mainwindow.h"
 #include "ui/ui.h"
 
-int main() {
+#include <QApplication>
+
+int main(int argc, char *argv[]) {
     // Create an instance of the DatabaseManager
     DatabaseManager data_base("audio_station_loop.db", "database/schema/schema.sql");
+    QApplication a(argc, argv);
+    MainWindow w;
+    w.show();
+    return a.exec();

```

Comment:
In addition, we should have more descriptive variable names so they explain what they do with the name